<ui version="4.0" >
 <class>CrashReporter</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Crash Reporter</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>%1 had a problem and crashed. We'll try to restore all your data when it restarts. To help us diagnose and fix the problem, you can send us a crash report.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Send crash report</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Add a comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
